packages/react-ui/storybook/stories/widgetsUI/CategoryWidgetUI.stories.js

```diff
@@ -42,63 +19,15 @@ const Template = (args) => {
   return <Widget {...args} />;
 };
 
-const LoadingTemplate = (args) => {
-  return (
-    <>
-      <Label variant='body1' mb={3}>
-        {'Limited width'}
-      </Label>
-      <ThinContainer>
-        <Widget {...args} />
-      </ThinContainer>
-
-      <Label variant='body1' mt={8} mb={3}>
-        {'Responsive'}
-      </Label>
-      <Widget {...args} />
-    </>
-  );
-};
-
-const data = [...Array(30)].map((_, idx) => ({
+const data = [...Array(200000)].map((_, idx) => ({
   name: `Category ${idx + 1}`,
   value: idx * 100
 }));
 
-const filteredData = data.slice(0, 4).map((cat, idx) => {
-  return { ...cat, value: idx % 2 === 0 ? null : cat.value };
-});
-
 export const Default = Template.bind({});
-const DefaultProps = { data };
```

Comment:
This simplification is against use cases visibility... Basically you're removing most of the stories to display this new case.... Please, let's think in a different way.

Reply:
Sure! Let me rollback this 👍